<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordPanel</class>
 <widget class="QWidget" name="RecordPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="timerLayout">
     <item>
      <widget class="QLabel" name="labelTimer">
       <property name="text">
        <string>Timer (seconds):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spTimerSeconds">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>86400</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="toolTip">
        <string>Recording duration in seconds. Set to 0 for continuous recording.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>(0 = continuous)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%v seconds</string>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QToolButton" name="pbStartOverwrite">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Start: Overwrite</string>
       </property>
       <property name="toolTip">
        <string>Start recording and overwrite existing files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbStopCapture">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Stop Capture</string>
       </property>
       <property name="toolTip">
        <string>Stop recording to all files</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxRawData">
     <property name="title">
      <string>Raw Binary Data</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="rawDataLayout">
      <item>
       <layout class="QHBoxLayout" name="rawFileLayout">
        <item>
         <widget class="QLabel" name="labelRawFile">
          <property name="text">
           <string>Raw File:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leRawFilename"/>
        </item>
        <item>
         <widget class="QToolButton" name="pbBrowseRaw">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="rawOptionsLayout">
        <item>
         <widget class="QCheckBox" name="cbRawAutoInc">
          <property name="text">
           <string>Auto Increment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbRawDisableBuffering">
          <property name="text">
           <string>Disable Buffering</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxParsedData">
     <property name="title">
      <string>Parsed CSV Data</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="parsedDataLayout">
      <item>
       <layout class="QHBoxLayout" name="csvFileLayout">
        <item>
         <widget class="QLabel" name="labelCsvFile">
          <property name="text">
           <string>CSV File:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leCsvFilename"/>
        </item>
        <item>
         <widget class="QToolButton" name="pbBrowseCsv">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="csvOptionsGrid">
        <item row="0" column="0">
         <widget class="QCheckBox" name="cbCsvAutoInc">
          <property name="text">
           <string>Auto Increment</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="cbWriteHeader">
          <property name="text">
           <string>Write Header</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cbWindowsLineEnding">
          <property name="text">
           <string>Windows Line Endings</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="cbInsertTimestamp">
          <property name="toolTip">
           <string>Insert a timestamp column as first column</string>
          </property>
          <property name="text">
           <string>Insert Timestamp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="cbStopOnClose">
          <property name="text">
           <string>Stop On Close</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="cbRecordWhilePaused">
          <property name="toolTip">
           <string>Continue recording to file even if the plot is paused</string>
          </property>
          <property name="text">
           <string>Record While Paused</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="cbCsvDisableBuffering">
          <property name="text">
           <string>Disable Buffering</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="csvFormatLayout">
        <item>
         <widget class="QLabel" name="labelSeparator">
          <property name="text">
           <string>Separator:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leSeparator">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>For TAB character enter \t</string>
          </property>
          <property name="text">
           <string>,</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDecimals">
          <property name="text">
           <string>Decimals:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spDecimals">
          <property name="toolTip">
           <string>Number of digits after comma</string>
          </property>
          <property name="value">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTimestampFormat">
          <property name="text">
           <string>Timestamp In:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbTimestampFormat"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>